### ISSUE TITLE
R: add posibility to change end time in GUI

### ISSUE THREAD

**Issue by tomxslo** [markwilkie/StreamCapture#83]
It would be nice to add duration / end time change in gui.

Today i would like to record:
FIFA U20 WC: Mexico vs England, Quarterfinal	World Football	Monday	13:00	15:00

Because its knock out phase it could go to extra time, so match will be longer than 2 hours. Wold be really nice to be able to change end time.

**Comment by markwilkie** [comment]
I agree that this would be good.  More specifically, I'd like to see:
- Ability to adjust start time in the UI by adding/subtracting minutes from the scheduled
- Ability to adjust end time in the UI by adding/subtracting minutes
